app/src/components/v-field-template/v-field-template.vue

```diff
@@ -78,6 +79,10 @@ export default defineComponent({
 
 		watch(() => props.modelValue, setContent, { immediate: true });
 
+		const grouplessTree = computed(() => {
+			return flattenFieldGroups(treeList.value);
+		});
```

Comment:
>  I'm assuming that that composable already starts of with a flat list before grouping it, so that would allow us to prevent having to ungroup the grouped list

I've wondered the same thing initially, but I assume we ultimately still need to use the current approach since the "flat list" doesn't contain related collections' fields? At least that's what I understood from useFieldTree, but I could be mistaken. 🤔

or maybe it'll require not using `topLevelFields` (since this is filtered without group), and a variation of `makeNode` that doesn't "transform" groups like it currently does (so that we'll still retain nested relational fields.

https://github.com/directus/directus/blob/4bab1946dcb366e28882b9d22ca42108eb0b66c4/app/src/composables/use-field-tree.ts#L69